app/pods/agenda/agendaitems/agendaitem/index/controller.js

```diff
@@ -19,6 +20,10 @@ export default class IndexAgendaitemAgendaitemsAgendaController extends Controll
       'page[size]': 1,
     });
     if (result.length) {
+      const isEditor = this.currentSession.isEditor;
+      const isDesignAgenda = this.agenda.isDesignAgenda;
+      const agendaItems = await this.agenda.agendaitems;
+      setAgendaItemsPriority(agendaItems, isEditor, isDesignAgenda);
       const neighbouringItem = result.firstObject;
```

Comment:
Deze code slaagt op het opnieuw berekenen/saven van de prioriteiten bij het verwijderen van een agendapunt.
In normale gevallen (PROD) worden er zelden agendapunten verwijderd, en dan meestal agendapunten die enkel voorkomen op een ontwerpagenda. In dat geval gaat deze code doen wat je verwacht.

Echter is er voor admins de mogelijkheid om een agendaitem te verwijderen dat zowel op de ontwerpagenda en op eerder goedgekeurde versies van de agenda staan. In dat geval gaan we enkel de huidig openstaande agenda aanpassen. Goedgekeurde agenda's gaan dan een gat hebben in de nummering.
Maar bij mijn weten gaat dit op PROD nooit gebeuren (zou alvast een bijzondere uitzondering moeten zijn)

Hiervoor verwacht ik geen changes, gewoon even neerschrijven dat er in dat geval vermoedelijk een manuele interventie zal nodig zijn van onze kant

Reply:
Specifieke acties die ik hiervoor moet ondernemen om de PR goedgekeurd te krijgen?